### ISSUE TITLE
Gradient method for conditional sampling

### ISSUE THREAD

**Issue by LuckyDC** [lucidrains/video-diffusion-pytorch#9]
Thanks for your effort in implementation. I did not find any code blocks using autograd package to compute gradient  as shown in Eq(6) of the paper. Have you implemented this technique? Or can you tell me which line corresponds to this technique. I am interested in its effectiveness. Thanks

**Comment by liangbingzhao** [comment]
have u figured out how to implement it? It seems this repo not implement it.

**Comment by Wang-Xiaodong1899** [comment]
Yes, it seems Eq(6) in the paper not be implemented here, right?

**Comment by juvekaradheesh** [comment]
Any updates?

**Comment by eloimoliner** [comment]
you may like to check this paper https://arxiv.org/abs/2209.14687
